packages/create-app.template.react.beta/templates/src/createClientAndAuth.ts.hbs

```diff
@@ -19,9 +19,9 @@ export default function createClientAndAuth() {
   const redirectUrl = import.meta.env.VITE_FOUNDRY_REDIRECT_URL;
   {{#if scopes}}
   const scopes = [
-    {{#each scopes}}
-      "{{this}}",
-    {{/each}}
+   {{#each scopes}}
+    "{{this}}",
+   {{/each}}
```

Comment:
I think this indentation is confusing to have the `#each` helper indented with one space in the context of this file (three spaces total), can we either keep this inline with the original indentation level (remove one space) or match the content (add one space)

```suggestion
  {{#each scopes}}
    "{{this}}",
  {{/each}}
```

Same applies for the all the rest of the templates!